### PR TITLE
Tests of RaceDao, RaceParticipationDao and TestDriveDao

### PR DESCRIPTION
**Pull request by theodesjardins**

### CODE REVIEW

File: formulaoneteam-persistence/src/main/java/cz/muni/fi/pa165/dao/characteristics/Inject.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package cz.muni.fi.pa165.dao.characteristics;
+
+public @interface Inject {
```
[the remark this answers is not in the document]
Comment (theodesjardins) [reply formulaoneteam-persistence/src/main/java/cz/muni/fi/pa165/dao/characteristics/Inject.java]: not one purpose, it was a mistake

File: formulaoneteam-persistence/src/test/java/cz/muni/fi/pa165/dao/race/RaceDaoTest.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        race.setLocation("location");
+        race.setId(id);
+
+        return race;
+    }
+
+    @Test
+    public void addRace() {
+        //given
+        Long id = Long.valueOf(100);
```
[the remark this answers is not in the document]
Comment (theodesjardins) [reply formulaoneteam-persistence/src/test/java/cz/muni/fi/pa165/dao/race/RaceDaoTest.java]: I write Long id = null ?

File: formulaoneteam-persistence/src/test/java/cz/muni/fi/pa165/dao/driver/DriverDaoTest.java [hunk 3]
```diff
@@ -1,53 +1,27 @@
...
 
         //then
-        AssertJUnit.assertNotNull(driverDao.findById(driver.getId()));
-    }
-
-    private Date createDate(int date, int month, int year) {
-        Calendar calendar = Calendar.getInstance();
-        calendar.set(year, month, date);
-        return calendar.getTime();
+        assertNotNull(driverDao.findById(driver.getId()));
```
Comment (Mrnda) [inline formulaoneteam-persistence/src/test/java/cz/muni/fi/pa165/dao/driver/DriverDaoTest.java]: Here you should also assert, that the found driver is equal to the driver you created.

File: formulaoneteam-persistence/src/test/java/cz/muni/fi/pa165/dao/race/RaceDaoTest.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        testRace = createRace(getDefaultRaceTitle());
+    }
+
+    @Test
+    public void addRace() {
+        //when
+        raceDao.add(testRace);
+
+        //then
+        assertNotNull(raceDao.findById(testRace.getId()));
```
Comment (Mrnda) [inline formulaoneteam-persistence/src/test/java/cz/muni/fi/pa165/dao/race/RaceDaoTest.java]: Same as above.

File: formulaoneteam-persistence/src/test/java/cz/muni/fi/pa165/dao/raceParticipation/RaceParticipationDaoTest.java [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        assertEquals(1, raceParticipation.getResultPosition());
+    }
+
+    @Test
+    public void addRaceParticipation() {
+        //when
+        raceParticipationDao.add(raceParticipation);
+
+        //then
+        assertNotNull(raceParticipationDao.findById(raceParticipation.getId()));
```
Comment (Mrnda) [inline formulaoneteam-persistence/src/test/java/cz/muni/fi/pa165/dao/raceParticipation/RaceParticipationDaoTest.java]: Same as above.

File: formulaoneteam-persistence/src/test/java/cz/muni/fi/pa165/dao/testDrive/TestDriveDaoTest.java [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+        testDrive = createTestDrive();
+    }
+
+    @Test
+    public void addTestDrive() {
+        //when
+        testDriveDao.add(testDrive);
+
+        //then
+        assertNotNull(testDriveDao.findById(testDrive.getId()));
```
Comment (Mrnda) [inline formulaoneteam-persistence/src/test/java/cz/muni/fi/pa165/dao/testDrive/TestDriveDaoTest.java]: Same as above

**Pull request merged**